From 639b83489c5dcb5e870233b910c1e5d77f62fcdf Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Tue, 22 Jun 2021 23:16:44 +0100
Subject: [PATCH 117/191] .bashrc: make error display nicer on the visual
 system

Do not move around. And do not use the dark red, ok?
---
 .bashrc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/.bashrc b/.bashrc
index 1761393..8dc0194 100644
--- a/.bashrc
+++ b/.bashrc
@@ -23,13 +23,13 @@ __ps1_show_exitcode () {
     # and for this have to use \001 \002 over \[ \] .
     # (Also see https://mywiki.wooledge.org/BashFAQ/053 .)
     if [ "$exitcode" -ne 0 ]; then
-	echo -ne '\001\033[01;41m\002'
-	echo -n "$exitcode"
+	echo -ne '\001\033[01;43m\002'
+	printf '%3d' "$exitcode"
 	echo -ne '\001\033[00m\002'
 	echo -n ' '
 	echo -ne '\001\033[01;32m\002'
     else
-	echo -ne '\001\033[01;32m\002'
+	echo -ne '  0 \001\033[01;32m\002'
     fi
 }
 
-- 
2.47.3

